packages/react-components/components/SettingsItem.tsx

```diff
@@ -134,8 +144,8 @@ export function SettingsItemInput({
           testID={testID}
           style={styles.input}
           inputStyle={[styles.innerInput, { color: inputColor }]}
-          onFocus={onFocus}
-          onBlur={onBlur}
+          onFocus={internalOnFocus}
```

Comment:
I think you can get away without the internalon* functions as if undefined is passed into TextInput it should ignore it itself. 

right?